Pod/Classes/PURLogStore.m

```diff
@@ -54,10 +51,15 @@ - (instancetype)initWithDatabasePath:(NSString *)databasePath
     self = [super init];
     if (self) {
         _databasePath = databasePath;
+        _databaseReadWriteCompletionQueue = dispatch_queue_create("PureeLogStoreReadWriteCompletion", NULL);
```

Comment:
Why did you change the completion queue to the subthread one?
If you changes, you need to add some lock-logics to PURBufferedOutput class.